src/docs.rs

```diff
@@ -112,10 +125,15 @@ pub fn generate_html(
             modules: modules_links.as_slice(),
             project_name: &project_config.name,
             page_title: &format!("{} - {}", name, project_config.name),
-            module_name: name,
+            module_name: name.clone(),
             project_version: "", // TODO
```

Comment:
If we have a version can we fill this?

Reply:
@michallepicki has done that on his branch. I can do the same or just leave it for his side.

Reply:
Should probably look differently than currently on my branch if version will be mandatory

Reply:
I've made the 'version' required and so I just pass it in here by reference. We'll see how it all looks together.